interpreter/cling/lib/Interpreter/BackendPasses.cpp

```diff
@@ -420,6 +462,7 @@ void BackendPasses::CreatePasses(int OptLevel, llvm::ModulePassManager& MPM,
                                  StandardInstrumentations& SI) {
 
   // TODO: Remove this pass once we upgrade past LLVM 19 that includes the fix.
+  MPM.addPass(UniqueInitFunctionNamePass());
```

Comment:
Oops. Didn't notice the comment. I'll move the comment down when I push to update LLVM tag.

Reply:
Not addressed yet AFACIT; the comment applies directly to `WorkAroundConstructorPriorityBugPass` and should move there...